sections/GPU.tex

```diff
@@ -17,14 +24,36 @@ \section{Graphics Processing Units}
 \vspace{-15pt}
 \end{figure}
 
-A simple representational comparison of a Central Processing Unit's - CPU - and a GPU's transistor usage is shown in \myref{image:GPUCPUimage}.
+A simple representational comparison of a CPU - and a GPU's transistor usage is shown in \myref{image:GPUCPUimage}.
 The GPU consists of more less powerful cores where as the CPU consists of fewer more powerful ones, the greater amount of cores allows for more computation power.
+But to be able to utilise this computation power a problem must able to be split up, so many or all the cores in the GPU is used.
 As of Q1 2015 an example of a modern high end desktop CPU is the Intel Haswell core i7 5960X which has 8 cores. \citep{puget}
 A contemporary high end GPU is the NVIDIA GTX 980 which has 2048 CUDA cores. \citep{techpowerup,gtx980}
 Due to their architectural differences the GPU allows for about 12 times more operations per second.
 
-This makes the GPU particularly useful for computation, even some which are not graphical. % Antagende/konkluderende?!?
+This makes the GPU particularly useful for large and complex computation, even some which are not graphical, if it is possibly to distribute the workload among the GPU cores.
+But the GPU has an overhead.
+This means that moving data and computations to the GPU cost a lot af time compared to running the computations on the CPU.
```

Comment:
Kan noget virkelig koste tid?
"This means that moving the data and then doing the computations on the GPU takes longer than running the computations on the CPU"